package.json

```diff
@@ -29,6 +29,7 @@
     "start-prod": "yarn build-prod && yarn serve-static",
     "start-prod-readable": "yarn build-prod-readable && yarn serve-static",
     "start-examples": "ws -d examples/ -s index.html -p 4242",
+    "start-docs": "gitbook serve ./docs-user",
```

Comment:
you need a `gitbook install` call here too. Otherwise if you delete `docs-users/node_modules` the command fails.
Maybe you can add a `init-docs` script for the step `gitbook install` so that you can call it from both places.